ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -47,7 +48,10 @@ struct IntegrationPointData final
     {
     }
 
-    typename BMatricesType::KelvinVectorType sigma, sigma_prev;
+    // For the consistency with the name of other mechanical process related
+    // processes,
+    // sigma is renamed as sigma_eff
+    typename BMatricesType::KelvinVectorType sigma_eff, sigma_prev;
```

Comment:
The comment goes into the commit message.
sigma_prev must be renamed too.

Reply:
No sun change anymore by passing class member via function argument.